Formula/libvirt.rb

```diff
@@ -62,4 +67,26 @@ def install
       end
     end
   end
+
+  test do
+    output = shell_output("#{bin}/virsh -v", 0)
```

Comment:
The `, 0` bit is assumed with `shell_output` unless it's told otherwise. Doesn't need stating explicitly.